<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="labelBusca">
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineBusca"/>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>440</y>
     <width>491</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Manipulação</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>304</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelNomeDisciplina">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Nome da Disciplina</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linenomeDisciplina"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>151</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="btnInserir">
       <property name="text">
        <string>Inserir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEditar">
       <property name="text">
        <string>Editar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExcluir">
       <property name="text">
        <string>Excluir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="labelNomeCurso">
       <property name="text">
        <string>Nome do Curso</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineNomeCurso"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableCurso_has_Disciplinas">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>521</width>
     <height>281</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>nomeCurso</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>nomeDisciplina</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
